pkg/apis/crd/v1alpha1/types.go

```diff
@@ -666,6 +670,33 @@ type FlowExporterFilter struct {
 	Protocols []string `json:"protocols,omitempty"`
 }
 
+// FlowExporterTLSConfig stores the TLS configuration used by the gRPC exporter and IPFIX
+// exporter with TLS transport.
+type FlowExporterTLSConfig struct {
+	// ServerName is used to verify the hostname on the returned certificate. If specified
+	// it will be included in the client's handshake (SNI) to support virtual hosting unless
+	// it is an IP address. If this field is omitted, no verification will be done.
```

Comment:
> If this field is omitted, no verification will be done.

This is incorrect. If this field is omitted the hostname used for certificate verification will be the value provided in spec.address.